roles/openshift_excluder/tasks/verify_upgrade.yml

```diff
@@ -0,0 +1,15 @@
+---
+# input variables
+# - repoquery_cmd
+# - openshift_upgrade_target
+- include: init.yml
+
+- include: verify_excluder.yml
+  vars:
+    #repoquery_cmd: repoquery_cmd
+    #openshift_upgrade_target: openshift_upgrade_target
+    excluder: "{{ item }}"
+  with_items:
+  - "{{ openshift.common.service_type }}-docker-excluder"
+  - "{{ openshift.common.service_type }}-excluder"
+  when: docker_excluder_on or openshift_excluder_on
```

Comment:
I think docker_excluder and openshift_excluder should be independent. So "docker_excluder_on or openshift_excluder_on" means any of excluder is set to true, then need verify both of excluders?